app/Http/Controllers/ReportsController.php

```diff
@@ -651,6 +655,13 @@ public function postCustom(Request $request)
                 $assets->whereBetween('assets.last_checkout', [$checkout_start, $checkout_end]);
             }
 
+            if (($request->filled('checkin_date_start')) && ($request->filled('checkin_date_end'))) {
```

Comment:
It's possible we might want to allow no end date, or maybe default the date to today? I could see that being an annoying process, having to pick that every time.

Reply:
Good thinking.

On the frontend, the end date is selected when the user selects the start date but they are able to clear the date and skip passing `checkin_date_end` to the backend.

I just pushed a commit that handles that situation.